changelog/seqera-enterprise/v24.2.mdx

```diff
@@ -0,0 +1,161 @@
+- Implemented custom launch container logic to specify a custom container to run Nextflow when submitting a workflow execution.
+- Tags propagation has been added to launch template, to enforce the resource label propagations to instances and volumes.
+
+### Runs
+
+- N/A
+
+### Datasets
+
+- N/A
+
+### Data Explorer 
+
+- Added a direct Data Explorer browse link to mounted data items, so that data used in a session can be inspected easily.
+
+### Data Studios
+ 
+ - Data Studios [custom environments](https://docs.seqera.io/platform/24.2/data_studios/custom-envs):
+    - Create custom analysis environments or link to publicly/privately hosted ECR container. 
+    - Custom environment build events are now added to the auditlog. This captures custom environment builds in the audit log to give you insight into how often, and which, custom environments are created. The events `data_studio_session_build_started` and `data_studio_session_build_failed` are added to the `tw_audit_log` table.
+- Data Studios dashboard:
+    -  New [dashboard](https://docs.seqera.io/platform/24.2/monitoring/dashboard#data-studios) provides information about Data Studios usage which can help you manage your resources.
+  - Hybrid compute enviroments now supported in Data Studios.
+    - Optionally define GPU allocation when creating a Data Studios session.
+  - NVMe support added.
+
+### Compute Environments
+
+- AWS Forge support has been added for Amazon Linux 2023. Previously AWS Forge was only able to create CEs compatible with AL2; Seqera now supports Amazon Linux 2023. 
+- The Google Life Sciences API will be deprecated in June 2025. A Google Life Sciences API deprecation notice has been added to the to Seqera Platform UI. 
+
+### Credentials 
+
+- Azure Batch service principal credentials and managed identities prevents spillover of credentials. 
+- Remove required validator from GitLab credentials password field.
+
+### Secrets
+
+- N/A
+
+### Participants 
+
+- N/A
+
+### Settings
+
+- Added flexibility for pipeline names in workspaces. This change forces name uniqueness at the workspace level for private workspaces while keeping the uniqueness at the organization level for shared workspaces.
+
+### User/authentication
+
+- N/A
+
+### General
+
+- Platform Progress store using Jedis in place of Redisson library. 
+- Use Jedis in place of Redisson in `DistributedMailService`.
+- Refactor reports file streaming over Agent connections directly to the browser.
+- JSON schema draft 2020-12 features inside parameters input form support. Includes: Nested parameters, "anyOf", "allOf", and "oneOf" input form validation support, "dependantSchemas" initial support, and "if-then-else" support.
```

Comment:
Improves support of json schema and prevents forms error in case of unrecognized validations